script.js

```diff
@@ -11,4 +13,293 @@ hamburger.addEventListener('click', toggleMenu);
+
+  const popupWindow = document.createElement('div');
+  popupWindow.setAttribute('id', 'popup-window');
+  popupWindow.classList.add('popup-section');
+
+  const popupCloseButton = document.createElement('button');
+  popupCloseButton.setAttribute('id', 'close-button');
+  popupCloseButton.classList.add('close-button');
+  popupCloseButton.innerHTML = '&times';
+  popupWindow.appendChild(popupCloseButton);
+
+  const popupImageDesk = document.createElement('img');
+  popupImageDesk.setAttribute('src', project.popupImageDesktop);
+  popupImageDesk.classList.add('desktop', 'popup-image');
+  popupWindow.appendChild(popupImageDesk);
+
+  const popupImageMob = document.createElement('img');
+  popupImageMob.setAttribute('src', project.popupImageMobile);
+  popupImageMob.classList.add('mobile', 'popup-image');
+  popupWindow.appendChild(popupImageMob);
+
+  const popupTitle = document.createElement('h3');
+  popupTitle.classList.add('popup-title');
+  popupTitle.innerHTML = project.popupTitle;
+  popupWindow.appendChild(popupTitle);
+
+  const popupTagsMob = document.createElement('ul');
+  popupTagsMob.classList.add('popup-items', 'mobile');
+  popupWindow.appendChild(popupTagsMob);
+
+  const popupTagsDesk = document.createElement('ul');
+  popupTagsDesk.classList.add('popup-items', 'desktop');
+  popupWindow.appendChild(popupTagsDesk);
+
+  for (let j = 0; j < project.languages.length; j += 1) {
+    const popupTag = document.createElement('li');
+    popupTag.innerHTML = project.languages[j];
+    popupTagsMob.appendChild(popupTag);
+  }
+  for (let j = 0; j < project.popupLanguages.length; j += 1) {
+    const popupTag = document.createElement('li');
+    popupTag.innerHTML = project.popupLanguages[j];
+    popupTagsDesk.appendChild(popupTag);
+  }
+
+  const popupDescription = document.createElement('p');
+  popupDescription.classList.add('popup-text');
+  popupDescription.innerHTML = project.description;
+  popupWindow.appendChild(popupDescription);
+
+  const popupButtonsContainer = document.createElement('div');
+  popupButtonsContainer.classList.add('popup-buttons');
+  popupWindow.appendChild(popupButtonsContainer);
+
+  const popupLiveLink = document.createElement('button');
+  popupLiveLink.setAttribute('id', 'popup__live__link__button');
+  popupLiveLink.setAttribute('type', 'button');
+  popupLiveLink.classList.add('button');
+  popupLiveLink.innerHTML = 'See Live';
+  popupButtonsContainer.appendChild(popupLiveLink);
+
+  const popupLiveLinkIcon = document.createElement('img');
+  popupLiveLinkIcon.setAttribute('src', 'images/see-live-icon.png');
+  popupLiveLinkIcon.setAttribute('alt', 'Live Link Icon');
+  popupLiveLink.appendChild(popupLiveLinkIcon);
+
+  const popupSourceLink = document.createElement('button');
+  popupSourceLink.setAttribute('id', 'popup__source__link__button');
+  popupSourceLink.classList.add('button');
+  popupSourceLink.innerHTML = 'See Source';
+  popupButtonsContainer.appendChild(popupSourceLink);
```

Comment:
Great work was done so far 👍🏼 👍🏼  , I want just to mention two small issues with these buttons on desktop the position is different it should be in the same level as the title , and on mobile increase the width of the button refer to the images below


![Capture d’écran du 2023-08-05 23-49-49](https://github.com/danielf-canon/Portfolio/assets/59707859/ddc76b5d-c4df-44d4-af26-f2dcfb9f8b29)
![Capture d’écran du 2023-08-05 23-49-29](https://github.com/danielf-canon/Portfolio/assets/59707859/5c567119-e6aa-442d-bfe7-f5f5a63cf159)
![Capture d’écran du 2023-08-05 23-48-09](https://github.com/danielf-canon/Portfolio/assets/59707859/59664f16-5a38-4c23-8587-0748a9de7668)
![Capture d’écran du 2023-08-05 23-47-30](https://github.com/danielf-canon/Portfolio/assets/59707859/3c643c69-167d-41d8-882c-77fc3f8270a7)